### PR TITLE
Update protocol to support SRT output

### PR DESCRIPTION
**Pull request by davidzhao**

### CODE REVIEW

File: packages/livekit-server-sdk/src/index.ts [hunk 1]
```diff
@@ -16,6 +16,8 @@ export {
   EncodingOptions,
   EncodingOptionsPreset,
   GCPUpload,
+  ImageCodec,
+  ImageFileSuffix,
```
Comment (lukasIO) [inline packages/livekit-server-sdk/src/index.ts:20]: just double checking that these two exports are the only thing we need for enabling SRT output?

Comment (davidzhao) [reply packages/livekit-server-sdk/src/index.ts:20]: these are unrelated to SRT. StreamProtocol should have SRT now though

File: packages/livekit-server-sdk/src/EgressClient.ts [hunk 2]
```diff
@@ -1,28 +1,31 @@
 // SPDX-FileCopyrightText: 2024 LiveKit, Inc.
 //
 // SPDX-License-Identifier: Apache-2.0
-import {
+import type {
```
Comment (davidzhao) [inline packages/livekit-server-sdk/src/EgressClient.ts:4]: @lukasIO I'm not sure if this is a good idea to change.. but my editor did it automatically.. should I revert?

Comment (nbsp) [reply packages/livekit-server-sdk/src/EgressClient.ts:4]: this is fine, there's no functional change, prettier enforces this for clarity

Comment (davidzhao) [reply packages/livekit-server-sdk/src/EgressClient.ts:4]: ok, give it a ✅ if it looks fine. we'll get it released

**Review by nbsp — APPROVED** [review]

**Pull request merged**